First version of Pettingzoo Multi-Agent Wrapper

TTomilin opened the pull request:
`pettingzoo_wrapper/base_pettingzoo_env.py` contains the multi-agent environment wrapper
`pettingzoo_wrapper/__init__.py` is the main entry point for creating envs
`examples/python/multi_agent.py` runs a few episodes with rendering for quick validation
`examples/python/train_multi_agent.py` uses the benchmarl library to train a multi-agent policy

mwydmuch commented on examples/python/multi_agent.py:
I think it would help to have pettingzoo in the name of the example

mwydmuch commented on pettingzoo_wrapper/base_pettingzoo_env.py:
```diff
@@ -0,0 +1,803 @@
+
+    # headless
+    game.set_window_visible(False)
+    game.set_sound_enabled(False)
```
I think sound and hud ideally should be loaded from the config file

mwydmuch commented on pettingzoo_wrapper/base_pettingzoo_env.py:
```diff
@@ -0,0 +1,803 @@
+            game.add_game_args("+viz_connect_timeout 45")
+        
+        game.init()
+        game.send_game_command("viz_respawn_delay 0")
```
Can be added to host arguments with + prefix

mwydmuch commented on pettingzoo_wrapper/base_pettingzoo_env.py:
```diff
@@ -0,0 +1,803 @@
+                })
+                steps += frames_per_step
+
+            elif cmd == "respawn":
```
What if there is respawn delay?

TTomilin commented:
I've addressed all the points you've raised. I am not sure about the respawn part. It seems to work fine as is. Needs further testing to see whether there are situations where issues can arise
